media/website/content/migration.md

```diff
@@ -30,6 +30,10 @@ Auditor REST APIs are changed as shown in the table below.
 
 `/records` endpoint handles multiple and bulk record operations such as inserting bulk records and querying multiple records.
 
+## Removed
+/get_[started/stopped]_since endpoint is removed due to the introduction of advanced query. The auditor client and pyauditor client still contains the get_started_since and get_stopped_since
```

Comment:
This is markdown, so maybe use `/get_[started/stopped]_since` instead of /get_[started/stopped]_since